joomla/joomla-cms
# Sync sql install and samples files

Pull Request for Issue #13981

### Summary of Changes

* Mysql Samples: add helpful comments for graphic diff app to compare correct lines.
* Mysql Samples: I have removed IGNORE from INSERT clause and it still work.
* Postgrsql install: add comments, move down/up a few tables in order to sync with mysql install
* Postgrsql samples: add comments and sync with mysql samples
* Mssql: sync with postgresql install and samples, replace all [] quotes to "".
* Mssql: replace INSERT ... SELECT UNION ALL ... to insert with values INSERT ... VALUES (...), (...);
Why I did it: https://blog.sqlauthority.com/2008/07/02/sql-server-2008-insert-multiple-records-using-one-insert-statement-use-of-row-constructor/

### Testing Instructions
Install each sample per database, install should success. 

On postgresql and mssql please check front page and backend (specially Component lists).

### Expected result
All as before, on postgresql and mssql after sync there may be more samples/items.

### Actual result
Joomla with samles on postgresql and mssql does not work. 
Take a look at backend->Components.


### Documentation Changes Required
None
